core/EE_Config.core.php

```diff
@@ -2412,7 +2412,7 @@ public function __construct()
         $this->default_STS_ID = EEM_Registration::status_id_pending_payment;
         $this->email_validation_level = 'wp_default';
         $this->show_pending_payment_options = true;
-        $this->skip_reg_confirmation = false;
+        $this->skip_reg_confirmation = true;
```

Comment:
since there's notes about this change, just drawing attention to it.  Is it _intentional_ that this boolean is flipped?

Reply:
yes this was intentional